modules/ROOT/pages/_partials/gs-writing-walkthroughs-proc.adoc

```diff
@@ -494,7 +494,7 @@ For example, to use the `CakePHP + MySQL (Ephemeral)` service in your custom {wa
 }
 ----
 +
-. Add your {walkthrough-name} to the cluster as described in xref:gs-publishing-walkthroughs-proc[Adding walkthroughs].
+. Add your {walkthrough-name} to the cluster as described in xref:gs-publishing-walkthroughs-proc[Adding {walkthrough-name}s].
 . Navigate the {walkthrough-name} to trigger the provisioning of the custom service, for example, the `cakephp-mysql-example` service.
 . Log in to the OpenShift console.
 . Navigate to the custom project in OpenShift.The project is named using the pattern `<username>-<walkthrough-id>`.
```

Comment:
Has the <walkthrough-id> also been renamed to solution pattern?

Reply:
Yep.  It has.